### ISSUE TITLE
Netstandard support

### ISSUE THREAD

**Issue by acidbless** [alexrainman/CarouselView#213]
Please add the support for the .netstandard projects.

**Comment by alexrainman** [comment]
Cannot be done. Abstractions depend on Xamarin.Forms package and it's not compatible with .Net Standard :(

**Comment by acidbless** [comment]
I rised this issue due to switching my project to Xamarin.Forms 2.4.0 Pre3. It now supports Netstandard projects. But now I cannot use your library directly. Of course I can create my own nuget package from the sources, but this is not the best solution.

**Comment by alexrainman** [comment]
Maybe i can ship a pre-release version. Wait for it.

**Comment by acidbless** [comment]
Thanks!

**Comment by lsuarez5280** [comment]
Just a note that Xamarin.Forms is now .NET Standard 1.0 compatible. Would be nice to see this reopened.

**Comment by alexrainman** [comment]
Guys, i already did this and i have it working for Android and iOS but theres no way i can make it work for UWP.

**Comment by ahoefling** [comment]
Is there a pre-release on NuGet or a branch we missed? I don't see one outside of master? Or did that go into the PR you submitted to Xamarin?

**Comment by alexrainman** [comment]
No, i havent put anything out yet. Havent updated the PR in a while neither. Have been busy.

**Comment by ahoefling** [comment]
Do you want to submit the PR or branch so others can use the code? I am more than happy to help you with it if there is work that still needs to be done before it is merged into master

**Comment by alexrainman** [comment]
Will do soon. Will let you guys know. Thanks
